### PR TITLE
Add ktlint

### LINKED ISSUE #111: Add ktlint
klint is a built-in formatted that contains the following features to help you to format and organize your code style.

https://github.com/pinterest/ktlint

### PR DESCRIPTION
**Pull request by DeKaN**
Also, fixed all current issues and added check to Github Actions workflow

Closes #111

### CODE REVIEW

**Comment by DeKaN** [comment]
@aritra-tech updated

File: .editorconfig
Comment (aritra-tech) [inline .editorconfig:1]: What is the use of the `.editorconfig` file?

Comment (DeKaN) [reply .editorconfig:1]: This file contains code style settings for the project and many editors and linters support it
Ktlint uses only `[*.{kt,kts}]` section

Comment (aritra-tech) [reply .editorconfig:1]: Okay got your point

Comment (DeKaN) [reply .editorconfig:1]: I took its content from my projects, you can update it for your code style, especially these preferences:

- `max_line_length`
- `ij_kotlin_allow_trailing_comma`
- `ij_kotlin_allow_trailing_comma_on_call_site`

**Review by aritra-tech — APPROVED** [review]
LGTM 👍

**Pull request merged**